src/elements/x-transactions/x-transactions.css

```diff
@@ -208,6 +213,63 @@ x-transactions.view-more .view-less {
 
 /* CASHLINKS */
 
+/* Filter */
+x-transactions:not(.filtered) .filtered,
+x-transactions.filtered .not-filtered {
+    display: none;
+}
+
+.transactions-filtered-header {
+    margin: -1em;
+    margin-bottom: 8px;
```

Comment:
It's using `em` because that is what the parent element's `padding` is defined in: `x-card { padding: 1em }`. And because the filtered-header needs to fill in the padding of the parent to appear flush with the card border, I am using the same measure.

Keep in mind that this style here is used for mobile screens. In the Safe, we have in the beginning created the CSS mobile-first. So the deskop style is defined further down in this stylesheet and is defined in `px`.